### ISSUE TITLE
Needs to remove mrb_define_method_vm

### ISSUE THREAD

**Issue by expeditiousRubyist** [mattn/mruby-sharedlib#9]
When compiling with the latest version of mruby, the following error was generated:
```
Cannot export mrb_define_method_vm: symbol not defined
collect2.exe: error: ld returned 1 exit status
```
When mrb_define_method_vm was removed from the .def file, the program compiles perfectly. Is this a function no longer in mruby? If so, you should probably update your .def file.

**Comment by mattn** [comment]
could you please send me a pull-request?

**Comment by expeditiousRubyist** [comment]
Alright, done.

**Comment by mattn** [comment]
closed by #10
